craig-rolleston/week5/mon/magic-8ball/app/controllers/secretnumber_controller.rb

```diff
@@ -0,0 +1,15 @@
+class SecretnumberController < ApplicationController
+
+    def secretnumber
+    end # secretnumber
+
+    def form
+        
+    end #form
+
+    def secretanswer
+        @number = rand(1..10).to_i
+    end# results
```

Comment:
you would need to add the `guessedNumber` here from params - if you look at the url for this, you can see what the person guessed:
Something like the below
`@guess = params[:guessedNumber]`

Reply:
Same also applies for rock paper scissors